_posts/technotes/2017-08-18-tesseract-16.md

```diff
@@ -0,0 +1,118 @@
+---
+name: tesseract
+layout: post
+title: "Tesseract and Magick: High Quality OCR in R"
+date: 2017-08-18
+authors:
+  - name: Jeroen Ooms
+    url: https://ropensci.org/about/
+categories:
+  - technotes
+tags:
```

Comment:
add subject area tags, like whatever you used in other tesseract post, maybe OCR, and topic tag that would be used for such a package if it went through onboarding